components/board/orangepi/data.go

```diff
@@ -28,4 +29,27 @@ var boardInfoMappings = map[string]genericlinux.BoardInformation{
 		},
 		Compats: []string{"xunlong,orangepi-zero2", "allwinner,sun50i-h616"},
 	},
+	op3lts: {
+		PinDefinitions: []genericlinux.PinDefinition{
+			{Name: "3", DeviceName: "gpiochip1", LineNumber: 122},
+			{Name: "5", DeviceName: "gpiochip1", LineNumber: 121},
+			// When we can switch between gpio and pwm, the line number would be 118.
+			{Name: "7", DeviceName: "gpiochip1", LineNumber: -1, PwmChipSysfsDir: "300a000.pwm", PwmID: 0},
```

Comment:
The PWM chip claims it has 2 lines (`cat /sys/class/pwm/pwmchip0/npwm`). Any idea what the second one does? edit: we talked in person. Neither of us can find any documentation for the second line. :shrug: